doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -1,13 +1,24 @@
 .. _autopts-linux:
 
-AutoPTS on Linux with nRF52 board
-##################################
+AutoPTS on Linux
+################
 
 Overview
 ========
 
 This tutorial shows how to setup AutoPTS client on Linux with AutoPTS server running on Windows 10
-virtual machine. Tutorial covers only nrf52840dk. Tested with Ubuntu 20.4 and Linux Mint 20.4.
+virtual machine. Tested with Ubuntu 20.4 and Linux Mint 20.4.
+
+There are three ways to test zephyr bluetooth host:
+
+- Testing Zephyr Host Stack on QEMU
+
+- Testing Zephyr Host Stack on native posix
+
+- Testing Zephyr combined (controller + host) build on Real hardware(such as nRF52)
+
+The first and second methods do not require actual hardware, for more information, please visit:
```

Comment:
Isn't this a bit misleading? To run the Zephyr host with QEMU or Native Posix, you'll still need a controller running on real hardware.

Reply:
Maybe this sentence should be deleted

Reply:
Or change as:

For the first and second methods, for more information, please visit......

Reply:
Or perhaps more explicitely (in case we add more lines): 
```
For running with QEMU or native posix, please visit 
https://docs.zephyrproject.org/latest/guides/bluetooth/bluetooth-tools.html?highlight=hci_uart#running-on-qemu-and-native-posix
```